CHANGELOG.md

```diff
@@ -8,6 +8,132 @@ The format is based on [these versioning and changelog guidelines](https://git.i
+
+#### [Toast](https://polaris.shopify.com/components/feedback-indicators/toast)
+
+The toast component is a non-disruptive message that appears at the bottom of the interface to provide quick, at-a-glance feedback on the outcome of an action.
+
+#### [Loading](https://polaris.shopify.com/components/feedback-indicators/loading)
+
+The loading component is used to indicate to merchants that a page is loading or an upload is processing.
+
+#### [ContextualSaveBar](https://polaris.shopify.com/components/forms/contextual-save-bar)
+
+The contextual save bar tells merchants their options once they have made changes to a form on the page. This component is also shown while creating a new object like a product or customer. Merchants can use this component to save or discard their work.
+
+#### [Autocomplete](https://polaris.shopify.com/components/forms/autocomplete)
+
+The autocomplete component is an input field that provides selectable suggestions as a merchant types into it. It allows merchants to quickly search through and select from large collections of options.
+
+### Enhancements
+
+- Changed `Form` to default the `method` to `post` in order to prevent accidental leaking of form details
+- Added support for boolean type on Choice error prop
+- Changed the esnext folder to contain individual, minimally transpiled JavaScript component files, as well as raw style and image assets
+- Added `onPortalCreated` prop to `Portal`
+- Improved consistency of `Badge` styling
+- Explicitly specifying `list-style` on `List`
+
+### Bug fixes
+
+- Fixed console error and used new ref syntax in `DataTable` (thanks to ([@duythien0912](https://github.com/duythien0912)) for the ([original issue](https://github.com/Shopify/polaris-react/issues/403)))
+- Fixed the ability to upload multiple files even when `allowedMultiple` prop is false
+- Fixed `Datatable` so it resizes with new content (thanks [@andrewpye](https://github.com/andrewpye) for the [original issue](https://github.com/Shopify/polaris-react/issues/387))
+- Fixed `RangeSlider` linear-gradient so it doesn't break the css build (thanks [@Ankitjasoliya](https://github.com/Ankitjasoliya) and [@nerfologist](https://github.com/nerfologist) for the [original issue](https://github.com/Shopify/polaris-react/issues/441))
+- Fixed issue in `Page`, where styling wasn't being applied correctly to Page Actions
+- Removed unnecessary bindings on `Modal`s `onClose` prop. [original issue](https://github.com/Shopify/polaris-react/issues/441))
+- Rearranged `primaryFooterAction` and `secondaryFooterAction` in `Card` (thanks [@sivakumar-kailasam](https://github.com/sivakumar-kailasam) for the [original issue](https://github.com/Shopify/polaris-react/issues/551))
+
+### Breaking changes
```

Comment:
Should breaking changes go before bug-fixes and enhancements? It feels more important